recipes/finetuning_openai_summaries.ipynb

```diff
@@ -0,0 +1,1723 @@
+{
```

Comment:
<p>4) from the datset</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>